validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -440,6 +440,12 @@ def stop(self):
                 pass
         if self._topology:
             self._topology.stop()
+    
+    def get_connection_status(self, connection_id):
+        if self._topology:
+            return self._topology.get_connection_status(connection_id)
+        else:
+            return None
```

Comment:
Just `pass`. It will return None at the exit of the function.

Reply:
> Explicit is better than implicit.

https://www.python.org/dev/peps/pep-0020/

:)